spec/factories/pieces.rb

```diff
@@ -16,4 +16,13 @@
     type 'Queen'
     association :game
   end
+
+  factory :rook do
+    color 'white'
+    active true
+    x_position 1
+    y_position 1
```

Comment:
You used the `y_position: 3` in all your tests, you could specify those coordinates here in the factory and save yourself the trouble of specifying them in each test